### DISCUSSION TITLE
Choose search engine when downloading cover image from internet

### DISCUSSION THREAD

**Discussion by nicoleritzer** [PaulWoitaschek/Voice#1409] [Ideas]
Would there be a way to choose a different source/search engine when downloading cover images from the internet?

Google images adds an overlay to its search results, which defeats the purpose. Imo the Bing image search UI is more suited to the "crop your own cover" method.

I especially like the "Sirin" app's use of Google Books to scrape both metadata and HQ cover images, but the GB library is limited. Being able to choose between several options would be great.

**Comment by PaulWoitaschek** [comment]
That's a great idea. I played with that idea a little and gave Duck Duck go a try. Then I thought: What if it's possible to intercept the URL loading? Turns out: It is possible.

https://user-images.githubusercontent.com/743462/183143628-098842fc-64d2-41ee-972f-323d9392cba5.mp4

**Reply by nicoleritzer** [reply]
Wow, I haven't seen this method before. It makes the process so much simpler and faster. Nice work.
